.github/workflows/performance-tests.yml

```diff
@@ -0,0 +1,61 @@
+name: Performance Tests
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build-and-test-performance:
+    name: ubuntu-latest, Node.js 16.x
+    
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: "16.x"
+          registry-url: "https://registry.npmjs.org"
+
+      - name: Use Python 3.x
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.x"
+
+      - name: Build
+        shell: bash
+        run: |
+          yarn --skip-integrity-check --network-timeout 100000 --ignore-engines
+          npx electron-replace-ffmpeg
+          npx electron-codecs-test
```

Comment:
I'm not sure these are necessary for the build? (they aren't called explicitly in the other CI/CD workflow)

Reply:
In January the main build did these steps in the testing flow, so I copied them here. The main build doesn't do these now, so I'll delete them. Thanks!